### DISCUSSION TITLE
Search shortcut should also be able to close search prompt

### DISCUSSION THREAD

**Discussion by JeppeKlitgaard** [espanso/espanso#1246] [Feature requests and Ideas]
Hey @federico-terzi!

I thought it would be neat if it was possible to close the search prompt by pressing the search shortcut again. Currently you would have to press `ESC` or click off the prompt.

When I accidentally open the prompt, it would be easier to simply input the search shortcut again. This also feels intuitive from a UX perspective in that you are 'toggling off' the prompt after toggling it on.

**Comment by federico-terzi** [comment]
Hey @JeppeKlitgaard,

Thank you for the feedback! This seems a nice thing to have, I'll investigate how hard it is to accomplish

Cheers :)
